### ISSUE TITLE
[Suggestion] portable not sailfish only ui

### ISSUE THREAD

**Issue by norayr** [sailfishos-applications/daily-comics#99]
#### DESCRIPTION
hello. any chance to have non sailfish, just generic linux version (sailfish is not the only widespread mobile linux anymore, probably it is one of the not widespread)

#### ADDITIONAL INFORMATION
amazfish does it, has two uis. so i use it on any devite supported by postmarketos or waemo leste.

**Comment by Olf0** [comment]
Yes, an alternative UI which does not use Silica (likely as an compile time option) is technically sure feasible.  IMO it should address Qt 5.15 (or 6.x).  It might be feasible to use some shared code, especially when addressing Qt 5.15 (i.e. not 6.x).  It also would be nice to support Qt on classic PC (Laptop, Desktop, Tablet etc.).  But these are design decisions which should be discussed before someone starts to implement this, because the situation which Qt versions are available in which ("mobile" or not) Linux distributions is likely to change until then.

Note that I am just maintaining this app and will not implement new features.  But if you (or someone else) poses a PR, I will sure review it and consider to integrate it, if it is working fine.
